packages/odyssey-react-mui/package.json

```diff
@@ -47,17 +49,22 @@
     "@types/jest-axe": "^3.5.1",
     "@types/react": "^17.0.30",
     "@types/react-dom": "^17.0.5",
+    "@types/yargs": "^17",
     "babel-plugin-import": "^1.13.5",
     "eslint": "^7.27.0",
+    "fs-extra": "10.1.0",
```

Comment:
Pretty sure we shouldn't be using `fs-extra`.

Is this something we can remove?

Reply:
Investigating as part of the follow up: https://github.com/okta/odyssey/pull/1791